ui_framework/components/table/_table.scss

```diff
@@ -63,13 +63,22 @@ $tableCellPadding: 7px 8px 8px;
 }
 
 .kuiTableRow {
+  transition: background $globalAnimSlightResistance $globalAnimSpeedSlow;
   &:hover {
     .kuiTableRowHoverReveal {
       display: inline-block;
     }
   }
 }
 
+.kuiTableRow-isBeingDeleted {
+  background: lighten($errorColor, 35%);
+}
+
+.kuiTableRow-isNew {
```

Comment:
@cjcenizal This is why I'd rather use mixins for the css classes, so no preference is allowed :)

I generally don't mind nested classes, but I didn't see much in your code so I kept it out. I'll switch it around if that's the norm.